examples/simple_example/main.tf

```diff
@@ -34,28 +34,31 @@ resource "google_project_service" "iam" {
   disable_on_destroy = "false"
 }
 
-data "google_compute_image" "jenkins_agent" {
-  project = google_project_service.cloudresourcemanager.project
-  family  = "jenkins-agent"
-}
+module "artifacts" {
+  source = "../../modules/artifact_storage"
 
-resource "google_storage_bucket" "artifacts" {
-  name          = "${var.project_id}-jenkins-artifacts"
-  project       = var.project_id
-  force_destroy = true
-}
+  project_id = var.project_id
+  jobs_count = 1
 
-data "local_file" "example_job_template" {
-  filename = "${path.module}/templates/example_job.xml.tpl"
+  jobs = [
+    {
+      name = "testjob"
+
+      builders = [
+        <<EOF
+<hudson.tasks.Shell>
+  <command>echo &quot;hello world from testjob&quot;
+  env &gt; build-log.txt</command>
+</hudson.tasks.Shell>
+EOF
+      ]
+    }
+  ]
 }
 
-data "template_file" "example_job" {
-  template = data.local_file.example_job_template.content
-
-  vars = {
-    project_id            = var.project_id
-    build_artifact_bucket = google_storage_bucket.artifacts.url
-  }
+data "google_compute_image" "jenkins_agent" {
+  project = google_project_service.cloudresourcemanager.project
+  family  = "jenkins-agent"
 }
 
 resource "google_compute_firewall" "jenkins_agent_ssh_from_instance" {
```

Comment:
These firewalls are included in the issue.

Reply:
I thought they were too trivial.